docs/reuse/apps/view-dashboards.md

```diff
@@ -0,0 +1,3 @@
+All dashboards have a set of filters that you can apply to the entire dashboard. Use these filters to drill down and examine the data to a granular level.
+- You can change the time range for a dashboard or panel by selecting a predefined interval from a drop-down list, choosing a recently used time range, or specifying custom dates and times. [Learn more](/docs/dashboards/set-custom-time-ranges/).
+- You can use template variables to drill down and examine the data on a granular level. For more information, see [Filter with template variables](/docs/dashboards/filter-template-variables/).
```

Comment:
@kevin-sumo @omid-mortazavi 
Should we add here below section
All next-gen apps contain metadata filter(_sourceCategory) which gives users the ability to change their scope and avoid creating multiple copies of the dashboards each with a different scope.

Reply:
> All next-gen apps
Not all, right? Eg. Otel apps don't scope.

If we wanna add something here, I'd maybe word it like this:
> Most next-gen apps allow for providing the "scope" at installation time. The scope is comprised of a key (`_sourceCategory` by default) and a default value for this key.
> Based on the user input, the app dashboards will be parameterized with a dashboard variable, allowing users to easily change the dataset queried by all panels.
> This eliminates the need to create multiple copies of the same dashboard with different quries for this purpose.